client/my-sites/stats/activity-log-banner/success-banner.jsx

```diff
@@ -95,23 +101,23 @@ class SuccessBanner extends PureComponent {
 				{
 					<p>
 						{ backupUrl
-							? translate( 'We successfully restored your site back to %s!', { args: date } )
-							: translate( 'We successfully created a backup of your site as of %s!', {
-									args: date,
-								} ) }
+							? translate( 'We successfully created a backup of your site to %s!', { args: date } )
+							: translate( 'We successfully restored your site back to %s!', { args: date } ) }
 					</p>
 				}
 				{ backupUrl ? (
 					<Button href={ backupUrl } onClick={ this.trackDownload } primary>
-						{ translate( 'Download your backup' ) }
+						{ translate( 'Download' ) }
 					</Button>
 				) : (
 					<Button href={ siteUrl } primary>
 						{ translate( 'View site' ) }
 					</Button>
 				) }
 				{ '  ' }
-				<Button onClick={ this.handleDismiss }>{ translate( 'Thanks, got it!' ) }</Button>
+				{ ! backupUrl && (
+					<Button onClick={ this.handleDismiss }>{ translate( 'Thanks, got it!' ) }</Button>
```

Comment:
This may have already existed, but I'm questioning if `Thanks, got it!` is a bit ambiguous. Could we have an editor maybe follow-up here and review this string? Please pardon me if this already went through review.

cc: @michelleweber (are there people who do Calypso editorial reviews like this?)